packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNode.ts

```diff
@@ -223,6 +223,8 @@ export class SummarizerNode implements IRootSummarizerNode {
 				},
 				// These errors are usually transient and should go away when summarize is retried.
 				retryAfterSeconds: 1,
+				// Retrying summarization once should be enough for the error to go away,
+				retryCount: 1,
```

Comment:
Should we be annotating the error thrown from completeSummaryCore with these retry props too?  For the default case for your flight

Reply:
I simplified the whole logic, so this change is removed.